subprojects/docs/src/samples/userguide/dependencies/disablingTransitiveDependencyResolutionForConfiguration/build.gradle

```diff
@@ -0,0 +1,22 @@
+apply plugin: 'java-library'
+
+repositories {
+    mavenCentral()
+}
+
+//START SNIPPET transitive-per-configuration
+configurations {
+    all {
```

Comment:
Better to write `configurations.all {}` maybe to clarify that `all` is a method and not a configuration name?

Reply:
It's another case of being able to achieve the same thing in different ways. We should strive to reduce it as much as we can. It's simply confusing users. I will change it as proposed.